src/core/filesystem.cc

```diff
@@ -1269,8 +1348,14 @@ Status
 DestroyFileFolder(const std::string& path)
 {
   FileSystem* fs;
-  RETURN_IF_ERROR(GetFileSystem(path, &fs));
-  return fs->DestroyFileFolder(path);
+  // If path represents local temporary file then must be deleted
+  if (path.rfind("/tmp/file", 0) == 0) {
+    static LocalFileSystem lfs;
+    fs = &lfs;
+    return fs->DestroyFileFolder(path);
```

Comment:
remove `fs` and directly call `return lfs.DestroyFileFolder(path);`. Otherwise looks good

Reply:
Done